test/issue-1228.cl

```diff
@@ -0,0 +1,13 @@
+// RUN: clspv -inline-entry-points -g -O0 %s -o %t.spv 
+
+typedef struct Baz {
+    float x;
+} Baz;
+
+static int foo(Baz baz) {
+	return 1;
```

Comment:
Why did you add this `return 1;`? If you really want to keep it, could we avoid mixing space and tab in the file?

Reply:
ah sorry I was doing some debugging and forgot to change it back..